RecoTracker/TkNavigation/plugins/CosmicNavigationSchool.cc

```diff
@@ -44,6 +44,7 @@ class dso_hidden CosmicNavigationSchool : public SimpleNavigationSchool {
   //FakeDetLayer* theFakeDetLayer;
   void linkBarrelLayers( SymmetricLayerFinder& symFinder) override;
   //void linkForwardLayers( SymmetricLayerFinder& symFinder); 
+  void establishInverseRelations() override {};
```

Comment:
this overlaps in the context lines with https://github.com/cms-sw/cmssw/pull/22765/files#diff-7e30cfbaef0271d4950b3a87fc88ca51R47
Given similarlity of changes, please update #22765 instead of making an alternative PR.